src/gettingstarted/template/create-project.md

```diff
@@ -0,0 +1,49 @@
+# Start with a template
+
+## Create a new project from a template
+
+In the previous step, you set up a free trial account with Platform.sh. Now you have access to the Platform.sh [management console](/administration/web.md).
+
+From here you can create projects, adjust account settings, and a lot more that you will explore throughout these Getting Started guides.
+
+<video controls autoplay loop>
+  <source src="/videos/management-console/create-project-mc.mp4" type="video/mp4">
+</video>
+
+Since you do not yet have any projects on Platform.sh, the first thing you will see when you sign in is a workflow for creating a new project.
+
+1. **Name your project**
+
+   Give your project a name like `My First Project` and then click `Next`.
+
+2. **Set code base**
+
+   Now you will be able to see the large collection of Platform.sh's supported templates. They are simple applications, but they could also be the starting place from which you could build something more interesting.
```

Comment:
This description is technically no longer true.  Suggested revision:

"Now you will be able to see the large collection of Platform.sh's supported templates.  There are several types of templates available, including simple language-specific examples, ready-to-use frameworks you can build upon, and fully installed applications you can start using immediately after installation."